packages/management/addons-manager/package.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: Package
+metadata:
+  name: addons-manager.tanzu.vmware.com.1.5.0
+  namespace: addons-manager
+spec:
+  refName: addons-manager.tanzu.vmware.com
+  version: 1.5.0
+  releaseNotes: "addons-manager 1.5.0"
+  licenses:
+    - "UNKNOWN"
+  template:
+    spec:
+      fetch:
+        - imgpkgBundle:
+            image: projects.registry.vmware.com/tkg/packages/core/addons-manager@sha256:31c3f27350b693fec042ddead8906c119870ec8a876b9ee3e698a8d5d9c6d8b7
+      template:
+        - ytt:
+            paths:
+              - config/
+        - kbld:
+            paths:
+              - "-"
+              - .imgpkg/images.yml
```

Comment:
We are missing images.yml for the package CR and config.

Reply:
maybe I should remove this line? @yharish991 has another comment above which ask me to remove `images.yml` in .imgpkg because it's not needed

Reply:
images.yml would be generated by running the `package-bundle` target at build time